news/2022-12-01-osumonthly-5.md

```diff
@@ -0,0 +1,184 @@
+
+While not being the longest nor fastest map osu!monthly has showcased, it is definitely a merit in and of itself. Coming in at 270 BPM, various rhythm changes, not to mention these timing changes are happening within 1/4 streams, and being over 5 minutes in nigh non-stop drumming action, this map definitely holds up to its name. Beware, both the replay and Peaceful make the map look like a walk in the park, but it is a sight to behold.
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/xdNJyMYHw0A" frameborder="0" allowfullscreen></iframe>
+
+Some other notable plays for November include [shinchikuhome](https://osu.ppy.sh/users/3174184)'s [DT 3x100 FC](https://osu.ppy.sh/scores/taiko/175425262) on [Lunaticon](https://osu.ppy.sh/beatmapsets/1727109#taiko/3529595), while this one is a Loved map it is no small feat to be done, but lets be real — shinchikuhome does no such thing as small feats. Also, [nurend](https://osu.ppy.sh/users/9905079)'s [nightcore 10x miss](https://osu.ppy.sh/scores/taiko/175201104) on [Central Nucleous](https://osu.ppy.sh/beatmapsets/695399#taiko/1473448) being the second highest combo and lowest miss nightcore enhanced play yet.
+
+### Notable osu!catch plays of the month
+
+*Section written by [Patatitta](https://osu.ppy.sh/users/16294034)*
+
+[Aryssiel](https://osu.ppy.sh/users/2389481) has broken yet another record, getting a DT FC on the top difficulty of the recently ranked map [Kick Back!](https://osu.ppy.sh/beatmapsets/1864673#fruits/3834979). This marks a new pp record for the mod, being the first 1000pp play on this category of play, and beating the previous record by over 50pp.
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/7NUzL8_lFh4" frameborder="0" allowfullscreen></iframe>
+
+There has been many other outstanding plays this month, but [Ttobas](https://osu.ppy.sh/users/2084568)' DT FC on the top difficulty of [Now on stage!!](https://osu.ppy.sh/beatmapsets/745838#fruits/1572198) has stood on top of them, showing once again their impressive convert and DT skills on this massively popular map.
+
+### Notable osu!mania plays of the month
+
+Roughly 24 hours passed since [stεganography](https://osu.ppy.sh/beatmapsets/1846639#mania/3793375) — osu!mania's most recent challenging chart — was released. That's the amount of time it took [bojii](https://osu.ppy.sh/users/10083439/mania) to submit the first Full Combo in its leaderboards.
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/npZLSLb6REA" frameborder="0" allowfullscreen></iframe>
+
+For those who remember the map since it's showcase in [MWC22's Grand Finals](/wiki/Tournaments/MWC/2022_4K#grand-finals), this is quite the feat given its pattern variety. It takes a jack of all trades and master of all to be able to achieve such a thing, and bojii brushed it off as it were just another normal chart.
```

Comment:
```suggestion
For those who remember the map since its showcase in [MWC22's Grand Finals](/wiki/Tournaments/MWC/2022_4K#grand-finals), this is quite the feat given its pattern variety. It takes a jack of all trades and master of all to be able to achieve such a thing, and bojii brushed it off as it was just another normal chart.
```

Reply:
past tense on the 2nd suggestion, I think it should stay as is